ci-automation/ci-config.env

```diff
@@ -82,3 +82,14 @@ GCE_IMAGE_NAME="flatcar_production_gce.tar.gz"
 GCE_GCS_IMAGE_UPLOAD="gs://flatcar-jenkins/developer/gce-ci"
 GCE_MACHINE_TYPE="${GCE_MACHINE_TYPE:-n1-standard-2}"
 GCE_PARALLEL="${PARALLEL_TESTS:-4}"
+
+# -- Azure --
+AZURE_HYPER_V_GENERATION="V1"
```

Comment:
I would rather like the test to use the logic from https://github.com/flatcar-linux/scripts/pull/273 to run the cl.internet test with v1 and otherwise use v2 only from now on. Or is there any reason to prefer v1?

Reply:
This is a good suggestion, makes sense to test v2 on all and v1 on the selected few.